packages/runtime/src/parser/parser.ts

```diff
@@ -915,8 +913,6 @@ export function parser(program: string, mode: ExecutionMode, interpreter: Interp
 		assertMaxCost(interpreter.gas, mode);
 	}
 
-	// TODO: check if we can calculate length in: https://www.pivotaltracker.com/story/show/176623588
```

Comment:
I dont think it solves the problem since we do not compile it here

Reply:
Please see here: https://github.com/algorand/go-algorand-sdk/blob/13da8508530b7dc2e9ae62766cccd0baaff0f4bd/logic/logic.go#L83

We don't need compiled version.